app/States/Shipments/ShipmentState.php

```diff
@@ -48,6 +48,15 @@ public static function config(): StateConfig
                         ],
                         Canceled::class
                     ],
+                    
+                    // Support for uncanceled transitions - defined individually
```

Comment:
Believe this can be an array like above in the future to allow transition from one->many